### ISSUE TITLE
False Positive ERROR: PH2, \defined('_JEXEC') or die;

### ISSUE THREAD

**Issue by anibalsanchez** [joomla-extensions/jedchecker#160]
Hi,

An extension with the following form of " \defined('_JEXEC') or die;" is generating false ERROR: PH2.

```php
//\defined('JPATH_PLATFORM') or die;
\defined('_JEXEC') or die;
```

@dryabov Please, could you check this issue.
